### PR TITLE
Tutorial: hint of the SwiftUI template.

### PR DESCRIPTION
**Pull request by VilleWitt**
No issue created for the issue.

### Short description 📝

> Gives a pointer to the SwiftUI template and indirectly to the existence of templates and scaffolding.

### Checklist ✅

- [✅] The code architecture and patterns are consistent with the rest of the codebase.

### CODE REVIEW

**Comment by fortmarek** [comment]
@all-contributors add @VilleWitt for docs

File: projects/docs/docs/tutorial/get-started.md [hunk 1]
```diff
@@ -35,10 +35,14 @@ And then run:
 tuist init --platform ios
 ```
 
-The `init` command will bootstrap an iOS application, which includes the `Info.plist` files, an `AppDelegate.swift,` a tests file, and a **`Project.swift` that contains the definition of the project.**
+The `init` command will bootstrap an iOS application, which includes the `Info.plist` files, an `AppDelegate.swift`, a tests file, and a **`Project.swift` that contains the definition of the project.**
 
 > If you have used the Swift Package Manager before, the `Project.swift` file is the equivalent to the `Package.swift`.
 
+:::note SwiftUI template
+`tuist init --platform ios --template swiftui` will bootstrap a SwiftUI project instead.
```
Comment (fortmarek) [inline projects/docs/docs/tutorial/get-started.md:43]:
```suggestion
`tuist init --platform ios --template swiftui` will bootstrap a SwiftUI iOS project instead.
```

**Review by fortmarek — APPROVED** [review]
Thanks @VilleWitt for the addition, it's a good idea to make more people aware of the template 🙌

**Review by danyf90 — APPROVED** [review]

**Comment by danyf90** [comment]
@VilleWitt is this in draft because you want to add more? Otherwise feel free to merge it 🚀

**Comment by VilleWitt** [comment]
No, that's it. :-) Just opened it as draft to not seem to "pushy" or uncooperative... Go ahead and merge (I'm not authorized to merge)

**Pull request merged**